conf.yaml

```diff
@@ -81,3 +82,67 @@ repository:
                     comment: |
                         {author}'s PR has reached the required number of votes
                         for merging.
+        -
+            name: Check merged PRs for review tag.
+            conditions:
+                - state: 'merged'
+                - created_at__ge: 2016
+            actions:
+                -
+                    action: assert_has_tag
+                    action_value: 'status/review'
+                    comment: |
+                        This PR was merged without a 'status/review' tag, please correct.
+        -
+            name: Check merged PRs for a kind tag.
+            conditions:
+                - state: 'merged'
+                - created_at__ge: 2016
+            actions:
+                -
+                    action: assert_has_tag__startswith
+                    action_value: 'kind/'
+                    comment: |
+                        This PR was merged without a 'kind/' tag, please correct.
+        -
+            name: Add status/review to new PRs.
+            conditions:
+                - state: 'merged'
+                - created_at__ge: '2016-01-01'
```

Comment:
There has to be a better way to do this. *Has* to. We *should* have access to the head commit ref of each branch, I'm guessing that combined with `git branch -r --contains <commit> | grep release_...` will tell us which PRs are associated with a given release. Given that information, I'd say we change the condition from `created_at` to `in_release`, and use the `next_milestone` above